includes/framework/QDbBackedSessionHandler.class.php

```diff
@@ -123,6 +123,15 @@ public static function SessionRead($id) {
 			if (!$result_row) // either the data was empty or the row was not found
 				return '';
 			$strData = $result_row['data'];
+
+			if(strstr($objDatabase->Adapter, 'PostgreSql')) {
```

Comment:
Since we don't do runtime analysis of the database, but only do that during codegen, @vaibhav-kaushal is right. Either we need to make this code go through the data model, or we need a special case for postgresql since it does not have good support for BLOB or binary columns.